pattern/constraint/thing/IsaConstraint.java

```diff
@@ -63,31 +58,10 @@ public boolean isExplicit() {
         return isExplicit;
     }
 
-    public void addHints(Set<Label> labels) {
-        typeHints.addAll(labels);
-    }
-
-    public void retainHints(Set<Label> labels) { typeHints.retainAll(labels); }
-
-    public void removeHint(Label label) {
-        typeHints.remove(label);
-    }
-
-    public void clearHintLabels() {
-        typeHints.clear();
-    }
-
-    public Set<Label> getTypeHints() {
-        return typeHints;
-    }
-
     @Override
     public void addTo(Traversal traversal) {
         // TODO: assert !(type.reference().isLabel() && typeHints.isEmpty());
```

Comment:
probably this assertion needs to move to wherever the type hints are now too?

Reply:
@maxbaxt you did not remove this assertion........

Reply:
You mean the comment? Deleted